wallets/metamask/test/cypress/batchTransfer.cy.ts

```diff
@@ -0,0 +1,19 @@
+before(() => {
+  cy.get('#deployERC1155Button').click()
+
+  cy.confirmTransaction().then(() => {
+    cy.wait(5000)
```

Comment:
@matstyler why wait? we have to avoid waits at all cost

Reply:
we do not have tools to avoid it at the moment, I will have to resolve it separately

Reply:
@matstyler from my understanding, this transaction has to be confirmed before we can execute next steps in this test.

I think we should add a new task for it => if specific flag is passed to confirmtransaction (waitfortx: true?), then it will automatically wait for tx to be confirmed before moving on.